### DISCUSSION TITLE
Different "the last version" in release notes

### DISCUSSION THREAD

**Discussion by fengli320** [git-for-windows/git#4933] [Q&A] [answered]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

git version 2.45.0.windows.1
cpu: i686
built from commit: b5d0511969ccd9ab86395c37e5a7619d8b4e7c32
sizeof-long: 4
sizeof-size_t: 4
shell-path: /bin/sh
feature: fsmonitor--daemon

```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 6.1.7601]
```

### Details

 - In the release notes of 2.45.0.windows.1:

Git for Windows for Windows **_v2.45_** is the last version to support for Windows 7 and for Windows 8

 - In the release notes of 2.44.0.windows.1:

Git for Windows for Windows **_v2.44_** is the last version to support for Windows 7 and for Windows 8

Questions:

 - Which version is **_the last_**  version to support for Windows 7 and for Windows 8?

 - Is "**_Git for Windows for Windows_**" a typo?

**Comment by dscho** [comment, marked as answer]
As per the [commit message](https://github.com/git-for-windows/build-extra/commit/1e346336dcf30e2553425b8aa2e5b3600b9dfed1) that added that newest version of the deprecation notice:

> Please note that v2.45.0 will still support Windows 7/8 due to the move to MSYS2 runtime v3.5.x being delayed (the MSYS2 project did not make that switch yet, for similar reasons, and also because the quick succession of the .0, .1, .2 and .3 releases raised doubts.).
